src/Shared.elm

```diff
@@ -92,6 +108,32 @@ init flags url key =
+
+                Just localSettings ->
+                    case D.decodeString decodeLocalSettings localSettings of
+                        Ok playlists ->
+                            playlists
+
+                        Err _ ->
+                            { localSettingsList = [ { name = "language", value = "English" }, { name = "player", value = "Auto" }, { name = "controls", value = "Kodi" }, { name = "Ignore article", value = "true" }, { name = "Album artists", value = "true" }, { name = "Focus playlist", value = "true" }, { name = "headers", value = "true" }, { name = "thumbs Up", value = "false" }, { name = "device name", value = "false" }, { name = "port", value = "9090" }, { name = "host", value = "localhost" }, { name = "Poll interval", value = "10 sec" }, { name = "settings level", value = "Standard" }, { name = "proxy", value = "false" }, { name = "nfo", value = "true" }, { name = "movie db", value = "" }, { name = "fanart tv", value = "" }, { name = "youtube", value = "" } ] }
+
+        decodedAddonSettings =
+            case flags.addonLocalSettings of
+                Nothing ->
+                    { localSettingsList = [ { name = "aac", value = "true" }, { name = "wma", value = "true" }, { name = "default", value = "true" }, { name = "sens", value = "true" }, { name = "input", value = "true" }, { name = "rtmp", value = "false" }, { name = "universal album", value = "true" }, { name = "genric album", value = "true" }, { name = "local", value = "true" }, { name = "universal artist", value = "true" }, { name = "genric artist", value = "true" }, { name = "music", value = "true" }, { name = "fanart", value = "true" }, { name = "imdb", value = "true" }, { name = "music brainz", value = "true" }, { name = "audio", value = "true" }, { name = "movie scraper", value = "true" }, { name = "movie database", value = "true" }, { name = "movie database python", value = "true" }, { name = "movie database tv", value = "true" }, { name = "tmdb", value = "true" }, { name = "joystick", value = "true" }, { name = "youtube", value = "true" }, { name = "addon", value = "true" }, { name = "weather", value = "true" }, { name = "english", value = "true" }, { name = "ui", value = "true" }, { name = "black", value = "true" }, { name = "dim", value = "true" }, { name = "certifi", value = "true" }, { name = "chardet", value = "true" }, { name = "idna", value = "true" }, { name = "python image", value = "true" }, { name = "python crypto", value = "true" }, { name = "requests", value = "true" }, { name = "six", value = "true" }, { name = "url lib", value = "true" }, { name = "version", value = "true" }, { name = "estouchy", value = "true" }, { name = "estuary", value = "true" }, { name = "chorus", value = "true" } ] }
```

Comment:
If you add a line break in these, elm-format will automatically format the rest of the type for you. I would recommend this for readability.